src/vs/platform/quickinput/browser/quickInput.ts

```diff
@@ -227,4 +230,72 @@ export class QuickInputService extends Themable implements IQuickInputService {
 			})
 		};
 	}
+
+	formatPinnedItems(accessor: ServicesAccessor, storageKey: string, quickPick: IQuickPick<IQuickPickItem>, callback: () => void): void {
```

Comment:
Originally the intent was that this would be in a file next to `quickInput.ts` and not a part of the service itself. I don't want to give the impression that this functionality will make it into API at all so we should keep it separate as a helper.

Reply:
ok will do